.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - run:
           name: bundle install
           command: |
-            bundle check || bundle install
+            bundle check || bundle install --jobs 4
```

Comment:
This is actually already taken care of by the `BUNDLE_JOBS: 4` environment variable above on line 9

Reply:
ooops didn't see that @mattbrictson